Make page-spread-center an alias for spread-none

mattgarrish opened the pull request:
The only concern I have now that I'm opening the PR is whether we should formally deprecate page-spread-center or if we should "discourage" its use. I don't know if anyone has used it that we'd affect by this change.

Otherwise, the PR itself removes all traces of page-spread-center except to deprecate and make it an alias and adds the centering prose to the rendition:spread none processing requirements.


- Reading Systems [preview](https://cdn.statically.io/gh/w3c/epub-specs/fix/issue-1929/epub33/rs/index.html)
- Reading Systems [diff](https://services.w3.org/htmldiff?doc1=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://w3c.github.io/epub-specs/epub33/rs/index.html&doc2=https://labs.w3.org/spec-generator/%3Ftype=respec%26url=https://cdn.statically.io/gh/w3c/epub-specs/fix/issue-1929/epub33/rs/index.html)


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/pull/1950.html" title="Last updated on Dec 10, 2021, 1:32 PM UTC (e472d62)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/epub-specs/1950/a6e27d4...e472d62.html" title="Last updated on Dec 10, 2021, 1:32 PM UTC (e472d62)">Diff</a>

mattgarrish committed:
deprecate page-spread-center and make it an alias for spread-none

iherman commented on epub33/rs/index.html:
```diff
@@ -1021,29 +1021,23 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
+					<p>Reading Systems MUST handle the <span id="page-spread-center"
+								><code>rendition:page-spread-center</code></span> property as though the EPUB Creator
+						has specified the <a href="#def-spread-none"><code>spread-none</code> property</a> on the
+							<code>itemref</code> element.</p>
```
Note that if `page-spread-center` is labeled as "deprecated" then the _MUST_ statement is to be made a bit more precise. Per the [definition of deprecated](https://w3c.github.io/epub-specs/epub33/core/#app-overview-unsupported) an RS _MAY_ implement it but not more. Ie, this statement should say something like "if the RS implements `page-spread-center` then it _MUST_…"

mattgarrish replied:
I'm not sure that works for aliasing, as it seems to suggest the reading system ignore it if it only supports spread-none even though it's an alias. Or that it can support the property independent of it being an alias (i.e., you support spread-none if you support rendition:page-spread-center, since only the former really exists).

Maybe "handle" is the wrong word here, as my intention was only to recognize it as the other property and do what that property requires (if supported).

What about reversing the relationship and dropping the handling, which flows from recognizing the value as an alias, so it becomes:

"Reading Systems that support the `spread-none` property MUST recognize the `rendition:page-spread-center` property as an alias for it, otherwise they MUST ignore the `rendition:page-spread-center` property."

iherman replied:
Yep, @mattgarrish, the reformulation works for me, but... don't we have to rework the content document as well? Namely, is the statement in  [§5.2.4](https://www.w3.org/TR/epub-33/#page-spread) on `rendition:page-spread-center` still correct? What is the intention there? Shouldn't it mean the same as the definition of `spread-none` in the previous section?

Cc @wareid

mattgarrish replied:
> Shouldn't it mean the same as the definition of spread-none in the previous section?

It should. With this update it now reads:

> rendition:page-spread-center [DEPRECATED]
> Use of the rendition:page-spread-center property is deprecated. EPUB Creators should use the spread-none property to disable spread behavior in Reading Systems.

I followed the pattern we used for rendition:spread portrait here. (But the deprecated label needs reformatting looking at how we applied it elsewhere.)

iherman replied:
Shoot. I should have realized that you also edited the content document... I will be silent now!

mattgarrish committed:
reword alias support per issue comment

mattgarrish committed:
reformat deprecated label

mattgarrish committed:
Merge branch 'main' into fix/issue-1929

shiestyle reviewed (changes requested):
I understand this change but most EPUB 3 files in Japan use 'page-spread-center' for 1st page as cover and we should avoid that EPUBCheck will recognize it as error and warning. 

So alias is acceptable but deprecated is not good label.

mattgarrish committed:
remove deprecated status from page-spread-center;
clarify reason for the property in the note on aliases;
additional minor cleanup of the page-spread-* section

mattgarrish committed:
Merge remote-tracking branch 'remotes/origin/main' into fix/issue-1929

# Conflicts:
#	epub33/core/index.html
#	epub33/rs/index.html

mattgarrish committed:
Merge branch 'fix/issue-1929' of https://github.com/w3c/epub-specs into fix/issue-1929

# Conflicts:
#	epub33/core/index.html
#	epub33/rs/index.html

mattgarrish committed:
Merge remote-tracking branch 'remotes/origin/main' into fix/issue-1929

# Conflicts:
#	epub33/core/index.html
#	epub33/rs/index.html

mattgarrish committed:
restore page-spread-center example

mattgarrish committed:
adjust change log date

shiestyle reviewed (approved):
Thanks, Matt. That's fine for me.
